hack/logging-dump.sh

```diff
@@ -239,10 +240,30 @@ get_elasticsearch_status() {
       oc exec $pod -- $curl_es/_cat/$cat_item?v &> $cluster_folder/$cat_item
     done
     oc exec $pod -- $curl_es/_cat/shards?h=index,shard,prirep,state,unassigned.reason,unassigned.description | grep UNASSIGNED &> $cluster_folder/unassigned_shards
+  else
+    oc exec $pod -- $curl_es/_search?sort=@timestamp:desc | python -mjson.tool > $cluster_folder/latest_documents.json
   fi
 
 }
 
+get_es_logs() {
+  local pod=$1
+  local logs_folder=$2/logs
+  echo -- POD $1 Elasticsearch Logs
+  if [ ! -d "$logs_folder" ]
+  then
+    mkdir $logs_folder
+  fi
+  if [[ $pod == logging-es-ops* ]]
+  then
+    path=/elasticsearch/logging-es-ops/logs
```

Comment:
Might want to consider deriving these from the final elasticsearch.yaml file that is rendered in the pod.